native/explorer/src/expressions.rs

```diff
@@ -1051,7 +1055,7 @@ pub fn expr_second(expr: ExExpr) -> ExExpr {
 pub fn expr_join(expr: ExExpr, sep: String) -> ExExpr {
     let expr = expr.clone_inner();
 
-    ExExpr::new(expr.list().join(sep.lit()))
+    ExExpr::new(expr.list().join(sep.lit(), true)) //TODO: ignore_nulls
```

Comment:
I would like @cigrainger and @billylanchantin opinion on this one (and we should write tests). Should join discard nulls? And what does it mean to ignore? does it mean they are removed or that they are considered an empty string?

In Elixir:

```
iex(2)> Enum.join [1, nil, 3], ","
"1,,3"
```

Reply:
https://docs.pola.rs/py-polars/html/reference/expressions/api/polars.Expr.list.join.html

If there is a `null` value, the whole output becomes `null`. 

>Ignore null values (default).
If set to False, null values will be propagated. If the sub-list contains any null values, the output is None.

Reply:
Thank you. I definitely prefer to ignore nulls then. Having the whole thing return null is surprising and hard to debug. So let's add tests and remove the comment and we are good to go on this one. :)

Reply:
Agree 100%

Reply:
Yep 100%